test/test_operations.py

```diff
@@ -1861,6 +1861,14 @@ def test_emb_bf16(self):
     emb_out = emb(index)
     assert emb_out.dtype == torch.bfloat16
 
+  def test_transpose_1d(self):
+
+    def test_fn(t1):
+      t1.t_()
```

Comment:
Is it better to test `t1.t_()` and `t1.t()` in two separate tests?

Reply:
yea, I will separate tests.